monkestation/code/datums/diseases/advance/symptoms/clockwork.dm

```diff
@@ -0,0 +1,222 @@
+						var/obj/item/organ/internal/stomach/battery/clockwork/organ = new()
+						//if(robustbits)
+							//organ.max_charge = 15000 //no longer exists(old bee code)
+						organ.Insert(Host, TRUE, FALSE)
+					else
+						var/obj/item/organ/internal/stomach/clockwork/organ = new()
+						organ.Insert(Host, TRUE, FALSE)
+					if(prob(40))
+						to_chat(Host, "<span class='userdanger'>You feel a stabbing pain in your abdomen!</span>")
+						Host.emote("scream")
+					return TRUE
+				if(ORGAN_SLOT_EARS)
+					var/obj/item/organ/internal/ears/robot/clockwork/organ = new()
+					if(robustbits)
+						organ.damage_multiplier = 0.5
+					organ.Insert(Host, TRUE, FALSE)
+					to_chat(Host, "<span class='warning'>Your ears pop.</span>")
+					return TRUE
+				if(ORGAN_SLOT_EYES)
+					var/obj/item/organ/internal/eyes/robotic/clockwork/organ = new()
+					if(robustbits)
+						organ.flash_protect = 1
+					organ.Insert(Host, TRUE, FALSE)
+					if(prob(40))
+						to_chat(Host, "<span class='userdanger'>You feel a stabbing pain in your eyeballs!</span>")
+						Host.emote("scream")
+					return TRUE
+				if(ORGAN_SLOT_LUNGS)
+					var/obj/item/organ/internal/lungs/clockwork/organ = new()
+					if(robustbits)
+						organ.safe_co2_max = 15
+						organ.safe_co2_max = 15
+						organ.n2o_para_min = 15
+						organ.n2o_sleep_min = 15
+						organ.BZ_trip_balls_min = 15
+						organ.gas_stimulation_min = 15
+					organ.Insert(Host, TRUE, FALSE)
+					if(prob(40))
+						to_chat(Host, "<span class='userdanger'>You feel a stabbing pain in your chest!</span>")
+						Host.emote("scream")
+					return TRUE
+				if(ORGAN_SLOT_HEART)
+					var/obj/item/organ/internal/heart/clockwork/organ = new()
+					organ.Insert(Host, TRUE, FALSE)
+					to_chat(Host, "<span class='userdanger'>You feel a stabbing pain in your chest!</span>")
+					Host.emote("scream")
+					return TRUE
+				if(ORGAN_SLOT_LIVER)
+					var/obj/item/organ/internal/liver/clockwork/organ = new()
+					if(robustbits)
+						organ.toxTolerance = 7
+					organ.Insert(Host, TRUE, FALSE)
+					if(prob(40))
+						to_chat(Host, "<span class='userdanger'>You feel a stabbing pain in your abdomen!</span>")
+						Host.emote("scream")
+					return TRUE
+				if(ORGAN_SLOT_TONGUE)
+					if(robustbits)
+						var/obj/item/organ/internal/tongue/robot/clockwork/better/organ = new()
+						organ.Insert(Host, TRUE, FALSE)
+						return TRUE
+					else
+						var/obj/item/organ/internal/tongue/robot/clockwork/organ = new()
+						organ.Insert(Host, TRUE, FALSE)
+						return TRUE
+				if(ORGAN_SLOT_EXTERNAL_TAIL)
+					var/obj/item/organ/external/tail/clockwork/organ = new()
+					to_chat(Host, "<span class='userdanger'>imagine you have a tail or not.</span>")
+					organ.Insert(Host, TRUE, FALSE)
+					return TRUE
+				if(ORGAN_SLOT_EXTERNAL_WINGS)
+					var/obj/item/organ/external/wings/functional/clockwork/organ = new()
+					to_chat(Host, "<span class='userdanger'>imagine you have wings or not.</span>")
```

Comment:
This creates a ton of chat spam, as you get it every time this triggers.

Reply:
yea im disabling it, already done.

Reply:
i honestly wish i knew how external organs work, but its coded in a weird way.